controllers/items.js

```diff
@@ -9,8 +9,9 @@ const ItemTagsModel = require("../models/items");
 
 const addTagsToItem = async (req, res) => {
   try {
-    const { itemId } = await ItemTagsModel.addTagsToItem(req.body);
-    return res.json({
+    const { itemId, wasSuccess } = await ItemTagsModel.addTagsToItem(req.body);
```

Comment:
We don't know what is `req.body`. This would cause trouble when migrating to Typescript and when refactoring.